test/index.js

```diff
@@ -547,6 +547,30 @@ describe('Nightmare', function () {
       value.should.equal('');
     })
 
+    it('should not type in a nonexistent selector', function*(){
+      var failed = false;
+      try {
+        yield nightmare
+          .goto(fixture('manipulation'))
+          .type('does-not-exist', 'nightmare');
+      } catch(e){
+        failed = true;
+      }
+      failed.should.be.true;
```

Comment:
.... out of scope if you were intent on changing any method that could be cleaned up anyway.  Rereading this, I think you may have intended changing only the rejection tests.